packages/flutter_tools/lib/src/commands/symbolicate.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+import 'package:native_stack_traces/native_stack_traces.dart';
+import 'package:native_stack_traces/src/elf.dart'; // ignore: implementation_imports
```

Comment:
Is there a github issue for getting the things the tool needs made part of the public API of this package?

Reply:
Yup, this is pending https://dart-review.googlesource.com/c/sdk/+/133720